### PR TITLE
Bump Whereabouts to v0.6.3

### PR DESCRIPTION
**Pull request by WebberHuang1118**
**Problem:**
Bump Whereabout to v0.6.3

**Solution:**
Bump Whereabout to v0.6.3

**Related Issue:**
https://github.com/harvester/security/issues/38

**Test plan:**
**Deploy to running Harvester**
- Build Harvester cluster from Master
- Enable Storage Network
- Creating a VM, and writing some data into VM
- Update Whereabout from v0.6.2 to v0.6.3
  - Enter pod `harvester-cluster-repo-xxxx`
    - $ zypper install -y tar gzip vim awk
    - $ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
    - $ chmod 700 get_helm.sh
    - $ ./get_helm.sh
    - $ cd /srv/www/htdocs/charts/
    - $ tar zxvf harvester-0.0.0-master-xxxx.tgz
    - $ cd harvester
    - Edit `dependency_charts/whereabouts/templates/cluster_role.yaml` as https://github.com/harvester/harvester/pull/4986/files#diff-68d6083d3b14b99745ebec834362446dfce30eaae514fc2d8a0ef62356164194
    - Edit `dependency_charts/whereabouts/templates/daemonset.yaml` as https://github.com/harvester/harvester/pull/4986/files#diff-f5b2c56ecf7b81e2c63f48afe93adffbf2c0a81570b673cb3365738e0bdee6f6
    - Edit `values.yaml` as https://github.com/harvester/harvester/pull/4986/files#diff-1e557ee14cc96a67bfde9c7a2f9b5099c4453a72d8cbe98dd3babfce02a1557b
    - $ rm -rf charts/whereabouts/
    - $ helm dependency update
    - $ cd ../
    - $ rm harvester-0.0.0-master-xxxx.tgz
    - $ tar zcvf harvester-0.0.0-xxxx.tgz harvester
    - Deleting pod fleet-agent-xxxx
    - Checking if daemoset harvester-whereabouts successfuly deployed image `ghcr.io/k8snetworkplumbingwg/whereabouts:v0.6.3-amd64`
    - Checking if Longhorn `backing-image-manager-xxxx` pods, `instance-manager-e-xxxx` pods, `instance-manager-r-xxxx` pods and `backing-image-ds-xxxx` pods (only appearing during uploading) have interface lhnet1 and its ip in the assigned range
  - Checking I/O access and data integrity in the VM
  - Restarting the VM, and checking I/O access and data integrity in the VM again

**Building New Harvester with this PR**
- Build ISO from this branch, and creating Harvester cluster with the ISO
- After Harvester brought up, Check if daemonset harvester-whereabouts successfully deploying image `ghcr.io/k8snetworkplumbingwg/whereabouts:v0.6.3-amd64`
- Enable Storage Network
- Checking if Longhorn `backing-image-manager-xxxx` pods, `instance-manager-e-xxxx` pods, `instance-manager-r-xxxx` pods and `backing-image-ds-xxxx` pods (only appearing during uploading) have interface lhnet1 and its ip in the assigned range
- Checking image upload
- Creating VM from uploaded image

**Note**
- The manifest updates are referred to the difference between Whereabout [v0.6.2](https://github.com/k8snetworkplumbingwg/whereabouts/commits/v0.6.2/doc/crds) and [v0.6.3](https://github.com/k8snetworkplumbingwg/whereabouts/commits/v0.6.3/doc/crds) in path `doc/crds`
- CVE scan for Whereabout v0.6.3 image [v0.6.3_CVE.txt](https://github.com/harvester/harvester/files/13961785/v0.6.3_CVE.txt)

### CODE REVIEW

File: deploy/charts/harvester/dependency_charts/whereabouts/templates/daemonset.yaml [hunk 1]
```diff
@@ -1,3 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: whereabouts-config
+  namespace: kube-system
+  annotations:
+    kubernetes.io/description: |
+      Configmap containing user customizable cronjob schedule
+data:
+  cron-expression: "30 4 * * *" # Default schedule is once per day at 4:30am. Users may configure this value to their liking.
```
Comment (starbops) [inline deploy/charts/harvester/dependency_charts/whereabouts/templates/daemonset.yaml]: Could we separate the ConfigMap manifest from daemonset.yaml into a new configmap.yaml? Though there's no difference, I think it's more well-organized in terms of structure. Thank you.

**Review by bk201 — APPROVED** [review]
LGTM, also test v1.2.1 -> master path with storage network.

File: deploy/charts/harvester/values.yaml [hunk 2]
```diff
@@ -499,7 +499,7 @@ whereabouts:
   image:
     pullPolicy: IfNotPresent
     repository: ghcr.io/k8snetworkplumbingwg/whereabouts
-    tag: "v0.6.2-amd64"
```
Comment (ibrokethecloud) [inline deploy/charts/harvester/values.yaml:502]: whereabouts has a multi-arch manifest, can we please change tag to just `v0.6.3` and that will ensure architecture specific image is automatically downloaded? I eventually need this for arm builds as well.

File: deploy/charts/harvester/dependency_charts/whereabouts/templates/daemonset.yaml (hunk 1 above)
Comment (WebberHuang1118) [reply deploy/charts/harvester/dependency_charts/whereabouts/templates/daemonset.yaml]: Updated per suggestion, thanks.

File: deploy/charts/harvester/values.yaml (hunk 2 above)
Comment (WebberHuang1118) [reply deploy/charts/harvester/values.yaml:502]: Updated per suggestion, thanks.

**Review by starbops — APPROVED** [review]
LGTM, thank you!

**Review by ibrokethecloud — APPROVED** [review]
lgtm. thanks.

**Pull request merged**